yb-voyager/config-templates/live-migration-with-fall-back.yaml

```diff
@@ -267,7 +291,19 @@ import-schema:
   ### Note - This is only valid for Oracle source database.
   ### Accepted values (true, false, yes, no, 1, 0) 
   ### Default - true
-  # enable-orafce: true   
+  # enable-orafce: true  
+
+  ### Run guardrails checks before the command is run. 
+  ### Note - This is only valid for PostgreSQL source database.
```

Comment:
This note is not valid for import-* commands, no? Pls check and remove it whererver not applicable.

Reply:
Yeah makes sense. Removed wherever not applicable